librz/util/calc.c

```diff
@@ -186,6 +186,23 @@ static RzNumCalcValue term(RzNum *num, RzNumCalc *nc, int get) {
 				return d;
 			}
 			left = Ndiv(left, d);
+		} else if (nc->curr_tok == RNCEXP) {
+			RzNumCalcValue d = prim(num, nc, 1);
+			RzNumCalcValue exp_left = left;
+			if (d.d - (int)d.d) {
+				RZ_LOG_WARN("floating point powers not yet supported")
+			}
+			if ((int)d.d > 0) {
+				for (int i = 1; i < (int)d.d; i++) {
+					left = Nmul(exp_left, left);
+				}
+			} else if ((int)d.d < 0) {
+				for (int i = -1; i >= (int)d.d; i--) {
+					left = Ndiv(left, exp_left);
+				}
+			} else {
+				left = Ndiv(left, exp_left);
+			}
```

Comment:
The logic code should go in a `Nexp` function, similarly to how it's done for all the other operations.